docs/source/certs_management.md

```diff
@@ -379,3 +379,77 @@ Certificate:
+  certfile: /crypto/tlsca/cert.pem
+  chainfile: /crypto/tlsca/chain.pem
+  keyfile: /crypto/tlsca/key.pem
+  name: tlsca
+  reenrollignorecertexpiry: true
+```
+
+Alternatively, `reenrollignorecertexpiry` can be also be set with an environment variable or Fabric CA server startup flag:
+
+* Set the FABRIC_CA_SERVER_CA_REENROLLIGNORECERTEXPIRY environment variable to true at CA startup
+* Start the CA with the flag --ca.reenrollignorecertexpiry=true
+
+Restart the Fabric CA servers.
+
+### Renew Peer Certificates
+
+For peer enrollment certificates and TLS certificates you can use either the `enroll` or `reenroll` function from the Fabric CA client. Use `reenroll` if you would like to reuse a private key by passing the `csr.keyrequest.reusekey` option and indicating the `mspdir` location of the existing private key (the private key is found in the <mspdir>/keystore directory):
+
+```
+fabric-ca-client enroll --<OTHER-OPTIONS>
+
+fabric-ca-client reenroll --csr.keyrequest.reusekey --mspdir <LOCATION-OF-IDENTITY'S-MSP-DIRECTORY-THAT-CONTAINS-THE-EXISTING-PRIVATE-KEY> --<OTHER-OPTIONS>
+```
+
+Replace the peer's enrollment certificate or TLS certificate.
+
+The peer's enrollment certificate is configured to be in the peer's `<peer.mspConfigPath>/signcerts` directory (if private key was not reused the new private key should go to `<peer.mspConfigPath>/keystore` directory).
+
+The peer's TLS certificate is configured to be in the peer's `<peer.tls.cert.file>` directory/file (if private key was not reused the new private key should go to `<peer.tls.key.file>`).
+
+Restart the peer.
+
+### Renew orderer certificates
+
+For orderer enrollment certificates you can also use the `enroll` or `reenroll` function, using the steps described above for peers.
+
+Orderer TLS certificates must be reenrolled with the option to reuse the private key. This is due to the orderer TLS certificates being configured in the system and application channels (orderer to orderer communication is verified using the public key in the configured TLS certificate). As of v1.4.9 or v2.2.1, orderer nodes verify a matching key rather than the entire configured TLS certificate, enabling orderer TLS certificate renewal without requiring a channel configuration update to update the certificate.
+
+Use `reenroll` and reuse the TLS certificate's private key by passing the `csr.keyrequest.reusekey` option and indicating the `mspdir` location of the existing private key (the private key is found in the <mspdir>/keystore directory):
+
+```
+fabric-ca-client reenroll --csr.keyrequest.reusekey --mspdir <LOCATION-OF-IDENTITY'S-MSP-DIRECTORY-THAT-CONTAINS-THE-EXISTING-PRIVATE-KEY> --<OTHER-OPTIONS>`
+```
+
+Replace the orderer's enrollment certificate or TLS certificate.
+
+The orderer's enrollment certificate is configured to be in the orderer's `<General.LocalMSPDir>/signcerts` directory (if private key was not reused the new private key should go to `<General.LocalMSPDir>/keystore` directory).
```

Comment:
what does "should go to" mean exactly - maybe rephrase to be explicit. one prior instance above also.